dev-util/radare2/radare2-2.2.0.recipe

```diff
@@ -30,7 +33,7 @@ PROVIDES="
 	cmd:rarun2$secondaryArchSuffix
 	cmd:rasm2$secondaryArchSuffix
 	cmd:rax2$secondaryArchSuffix
-	lib:libr2$secondaryArchSuffix = 2.0.1
+	lib:libr2$secondaryArchSuffix = $libVersionCompat
```

Comment:
There is no compat needed here. Please check the SONAME when doing such changes.
Also https://abi-laboratory.pro/tracker/timeline/radare2/

Reply:
Oops, sorry @korli and @extrowerk. I should have mentioned that I had not checked this myself. (I won't forget to mention it next time, if I don't check it.)
So thanks @korli for spotting this.

@extrowerk, obviously, the fix is easy: you just need to use `$libVersion` in this line, instead of `$libVersionCompat`.